my-app/src/App.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
```

Comment:
Consider separating third-party imports from the project's ones.

Reply:
/fxd